doc/changes/changes_0.1.0.md

```diff
@@ -9,6 +9,7 @@ t.b.d.
 ## Bug Fixes
  
  - #18: Fixed network connection 
+ - #51: Fixed network connection
```

Comment:
Why are there two issues with the exact same description?

Reply:
We had this issue before, thought that we fixed it, but it was back again. Unfortunately, we can't really test this automatically at the moment.